### PR TITLE
task sorting based on assignee

### PR DESCRIPTION
**Pull request by ShevchukTania**
**STATUS**
READY

**DESCRIPTION**

**CHECK LIST**
- [x] specs added
- [x] specs passed
- [x] code coverage >90%
- [x] DB indexes added if needed
- [x] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [x] "let!" is used in specs

image

image

### CODE REVIEW

File: app/assets/javascripts/account/sort_tasks_assignee.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+$( document ).on('turbolinks:load', function() {
+  var positions = $('.tasks').children('.level').map(function() {
```
Comment (BohdanBusko) [inline app/assets/javascripts/account/sort_tasks_assignee.js]: you can write like $('.tasks .level').map....

File: app/assets/javascripts/account/sort_tasks_assignee.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    return $(this).data('task-id');
+  }).get();
```
Comment (BohdanBusko) [inline app/assets/javascripts/account/sort_tasks_assignee.js:4]: why get()? it`s doesn`t work without it?

File: app/assets/javascripts/account/sort_tasks_assignee.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  $( ".sort" ).click(function() {
+    var userInfo = [];
+    $('.tasks').children('.level').each(function() {
```
Comment (BohdanBusko) [inline app/assets/javascripts/account/sort_tasks_assignee.js]: and here to

File: app/assets/javascripts/account/sort_tasks_assignee.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      var assigneeId = $(this).data('assignee-id');
+      if (assigneeId != false) {
```
Comment (BohdanBusko) [inline app/assets/javascripts/account/sort_tasks_assignee.js:10]: you can write if(assigneeId)

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**